### DISCUSSION TITLE
Can we use TurboRepo and TurboPack for exiting CRA apps?

### DISCUSSION THREAD

**Discussion by amarg26** [vercel/turborepo#2840] [Ideas]
Can we Migrate from CRA to TurboRepo and TurboPack for existing projects?

If Yes, Can we have the steps, please?

**Comment by anthonyshew** [comment]
Tuborepo, yes!

Turbopack, however, is using Next.js as a proving ground to validate our approach and server the 850,000+ Next.js developers today. For the future, we have some ideas about making Turbopack a general purpose bundler.
